WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -180,41 +181,43 @@ public void onBindViewHolder(final SiteViewHolder holder, int position) {
                 && !getItem(position + 1).isRecentPick;
         holder.divider.setVisibility(showDivider ?  View.VISIBLE : View.GONE);
 
-        holder.itemView.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                int clickedPosition = holder.getAdapterPosition();
-                if (isValidPosition(clickedPosition)) {
-                    if (mIsMultiSelectEnabled) {
-                        toggleSelection(clickedPosition);
-                    } else if (mSiteSelectedListener != null) {
-                        mSiteSelectedListener.onSiteClick(getItem(clickedPosition));
+        if (mIsMultiSelectEnabled || mSiteSelectedListener != null) {
```

Comment:
What was the reason for adding the check for `(mIsMultiSelectEnabled || mSiteSelectedListener != null)` here before assigning the click listener? I tried it out and it works fine, but it *could* introduce the problem of the click listener not being assigned if the view is created before `mSiteSelectedListener` has been assigned.